doc/swish/event-mgr.tex

```diff
@@ -232,3 +232,36 @@
 message from \var{owner}.
 
 \var{owner} is a process.
+
+\defineentry{informative-exit-reason?}
+\begin{procedure}
+  \code{(informative-exit-reason? \var{x})}
+\end{procedure}
+\returns{} boolean
+
+The \code{informative-exit-reason?} procedure returns \code{\#t} if \var{x} is
+a fault condition containing a continuation or containing a reason other
+than \code{normal} or \code{shutdown}.
+Otherwise it returns \code{\#t} if \var{x} is
+not \code{normal} or \code{shutdown}.
+
+\defineentry{normalize-exit-reason}
+\begin{procedure}
+  \code{(normalize-exit-reason \var{r} \opt{\var{e}})}
+\end{procedure}
+\returns{} \var{reason} and \var{details}
+
```

Comment:
What happens when \var{e} is not specified?

Reply:
Good catch. I've added some text that may help.